installer/manifests/keptn/templates/resource-service.yaml

```diff
@@ -38,6 +38,10 @@ spec:
           type: RuntimeDefault
         {{- end }}
       terminationGracePeriodSeconds: {{ .Values.resourceService.gracePeriod | default 60 }}
+      initContainers:
+        {{- include "keptn.initContainers.wait-for-nats" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-mongodb-datastore" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-shipyard-controller" . | nindent 8 }}
```

Comment:
There should not be any dependency here, am I right @bacherfl ?

Reply:
I removed the init containers here for now